### PR TITLE
[stable/wordpress] Replace lusotycoon/apache-exporter with bitnami/apache-exporter

### PR DESCRIPTION
**Pull request by marcosbc**
Signed-off-by: Marcos Bjoerkelund <marcos@bitnami.com>

#### What this PR does / why we need it:

Bitnami now builds an Apache Exporter based on the Lusitaniae/apache_exporter project. This PR updates this chart to use that container image.

#### Which issue this PR fixes
None

#### Special notes for your reviewer:
None
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/wordpress/README.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ The following table lists the configurable parameters of the WordPress chart and
 | `podAnnotations`                 | Pod annotations                                                               | `{}`                                                         |
 | `metrics.enabled`                | Start a side-car prometheus exporter                                          | `false`                                                      |
 | `metrics.image.registry`         | Apache exporter image registry                                                | `docker.io`                                                  |
-| `metrics.image.repository`       | Apache exporter image name                                                    | `lusotycoon/apache-exporter`                                 |
+| `metrics.image.repository`       | Apache exporter image name                                                    | `bitnami/apache-exporter`                                 |
 | `metrics.image.tag`              | Apache exporter image tag                                                     | `v0.5.0`                                                     |
```
Comment (tompizmor) [inline stable/wordpress/README.md]: Is v0.5.0 the proper tag?

File: stable/wordpress/values-production.yaml [hunk 2]
```diff
@@ -339,7 +339,7 @@ metrics:
   enabled: true
   image:
     registry: docker.io
-    repository: lusotycoon/apache-exporter
+    repository: bitnami/apache-exporter
     tag: v0.5.0
```
Comment (tompizmor) [inline stable/wordpress/values-production.yaml]: is this tag right?

Comment (marcosbc) [reply stable/wordpress/values-production.yaml]: Fixing it.

File: stable/wordpress/README.md (hunk 1 above)
Comment (marcosbc) [reply stable/wordpress/README.md]: Good point! I will fix it.

**Review by tompizmor — COMMENTED** [review]
/hold

**Pull request merged**